src/igvfd/schemas/measurement_set.json

```diff
@@ -269,6 +269,22 @@
                 "appscript": "[\"ENSMUSG00000044487\", \"john-doe:gene01\"]",
                 "igvf_utils": "ENSMUSG00000044487, john-doe:gene01"
             }
+        },
+        "functional_assay_mechanisms": {
+            "title": "Functional Assay Mechanisms",
+            "description": "The biological mechanism(s) this functional assay measures. For example, VAMP-seq (MultiSTEP) measures variant effects on protein carboxylation and secretion.",
```

Comment:
Maybe change mechanisms to processes? In GO terms we use the word Biological process so maybe it is more equivalent. "The biological processes measured by this functional assay. For example, a VAMP-seq (MultiSTEP) assay measures the effects of variants on protein carboxylation and secretion processes."